Formula/ehco.rb

```diff
@@ -21,27 +22,33 @@ class Ehco < Formula
 
   depends_on "go" => :build
 
+  uses_from_macos "netcat" => :test
+
   def install
-    system "go", "build", *std_go_args, "cmd/ehco/main.go"
+    ldflags = %W[
+      -s -w
+      -X github.com/Ehco1996/ehco/internal/constant.GitBranch=master
```

Comment:
Since it checks out a tag and not a branch, wouldn't it be better to call this `HEAD` ?

Reply:
thanks for review, maybe i can change `GitBranch` to `HEAD` next time i release ehco

Reply:
Yes, I was just setting this to match how upstream sets this variable.